allegroTest/allegroTest/Ship.cpp

```diff
@@ -81,6 +81,16 @@ But yes, rand() is pretty poor*/
 	}
 }
 
+void Ship::accelerate(bool accelerate)
+{
+	if(accelerate && m_speed<m_maxSpeed)
+	{ ++m_speed; }
+	else if(m_speed>0)
+	{ --m_speed; }
```

Comment:
Does this mean you have changed the physics of flying? So it slows down as soon as you release the accelerate(up) key?